pkg/collector/corechecks/servicediscovery/apm/detect.go

```diff
@@ -38,7 +38,7 @@ const (
 	Injected Instrumentation = "injected"
 )
 
-type detector func(pid int, args []string, envs envs.Variables, contextMap usm.DetectorContextMap) Instrumentation
+type detector func(dc usm.DetectionContext) Instrumentation
```

Comment:
Maybe call the parameter `context` or `ctx` instead of `dc` which is not so transparent?

Reply:
done